frontend/src/container/tables/Tables/FavoriteTable/FavoriteTable.styl

```diff
@@ -0,0 +1,17 @@
+.favoriteTable__row
+  color fontColor
+  &:hover
+    color fontColor
+  &__link
+    z-index 1
+    position absolute
+    // NOTE - 2022-11-30 - 75px is the 'listIconWidth' variable, variables are not working in calc
```

Comment:
Missing your initials here -> `// NOTE - ML - 2022-11-30 - 75px is the 'listIconWidth' variable, variables are not working in calc`
Also you should add a reference to where you can find this variable and add a comment on the other side to say where we have to modify the code when we modify this variable
For example (on the other side):
`// NOTE - ML - 2022-11-30 - if you modify this variable you will have to modify "FavoriteTable.styl"`